src/exoscale/compute/api/payload.clj

```diff
@@ -81,4 +81,5 @@
          payload (-> (sanitize-lists params)
                      (assoc :apiKey api-key :response "json")
                      (merge (expiry/args ttl)))]
-     (assoc payload :signature (sign (query-args payload) api-secret)))))
+     (cond-> payload
+       (false? (str/blank? api-secret)) (assoc :signature (sign (query-args payload) api-secret))))))
```

Comment:
I propose to replace `false?` by `not`. Really a nitpick 
```suggestion
     (cond-> payload
       (not (str/blank? api-secret))
       (assoc :signature (sign (query-args payload) api-secret))))))
```

Reply:
`str/blank?` is a true predicate unlike some other clojure ones and always returns a boolean, `false?` is safe here.

We often use `not` and `some?` to test for truthyness instead of `true?` and `false?` because a lot of Clojure functions which look like predicates (the ones ending with `?`) return `nil` in some cases.

I agree with the layout proposal from @bcachet breaking the condition and the action on two lines makes things a bit more legible.